.circleci/config.yml

```diff
@@ -46,7 +46,8 @@ jobs:
       - run:
           name: Dependencies
           command: |
-            sudo apt-get install -y  rsync ninja-build ccache
+            sudo apt-get update
+            sudo apt-get install -y  rsync ninja-build ccache python-pip
```

Comment:
Honestly I don't know much about CircleCI, but it says:
> Package python-pip is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source
However the following packages replace it:
  python3-pip

At https://app.circleci.com/pipelines/github/InsightSoftwareConsortium/ITK/3465/workflows/74239fd7-7453-43de-af8a-efdb2ed086e4/jobs/10221/parallel-runs/0/steps/0-106

Reply:
Let's try python3-pip